.github/workflows/lint.yml

```diff
@@ -20,4 +20,4 @@ jobs:
       - name: Prettier code.
         run: yarn prettier
       - name: Find dead code
-        run: yarn find-deadcode
+        run: test "$(yarn find-deadcode | tee "$(tty)" | wc -l)" -lt 4
```

Comment:
Why we need this ?

Reply:
As I've explained in the commit message: By piping the output of ts-prune into `sed`, we no longer get the behavior that any error makes the workflow fail. This command essentially restores that behavior